docs/ios/v2/how--to-guides/interact-with-room/room/session-store.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Session Store (Alpha)
+nav: 7.24
+---
+
+Session store is a shared realtime key-value store that is accessible by everyone in the room. Consider it as additional data associated with the session at the top level. Session store can be utilized to implement features such as pinned text, spotlight (which brings a particular peer to the center stage for everyone in the room), and more.
+
+The session store persists throughout the session and is cleared when the last peer leaves the room.
+
+<Note>
+💡 Session Store vs Peer Metadata
+
+While peer metadata is associated with individual peers and each peer can have their own metadata, session store remains the same for every peer in the room.
+
+</Note>
+
+## How to use
+
+Session store is only available after user joins the session. To obtain a reference to the session store, you need to override the `func on(sessionStoreAvailable store: HMSSessionStore)` function of the `HMSUpdateListener` protocol. Once you get the reference to the store you can save it in your app for accessing later.
```

Comment:
Session store is only available after user joins the session. - is this line needed anymore?

Reply:
does not hurt to have it stated explicitly